lib/editing/editor.dart

```diff
@@ -126,7 +126,18 @@ class Position {
 }
 
 abstract class CodeCompleter {
-  Future<List<Completion>> complete(Editor editor);
+  Future<Completions> complete(Editor editor);
+}
+
+class Completions {
+
+  List<Completion> completionList;
```

Comment:
@devoncarew  Tried that now, but then suddenly `completions.completions` seems strange to me. Still to change it ?